input/pagecontent/4-6-fhircast-event-content-sharing.md

```diff
@@ -19,7 +19,7 @@ Structured information may be added, changed, or removed quite frequently during
 
 #### Example Use Case
 
-A frequent scenario which illustrates a diagnostic report centered workflow involves an EHR, an image reading application, a reporting application, and an advanced quantification application.  The EHR, image reading application, and reporting application are authenticated and subscribed to the same topic using a FHIRcast Hub with the EHR establishing a patient context, see messages 1 through 7 in the below sequence diagram.
+A frequent scenario which illustrates a diagnostic report centered workflow involves an EHR, an image reading application, a reporting application, and an advanced quantification application.  The EHR, image reading application, and reporting application are authenticated, authorized and subscribed to the same topic using a FHIRcast Hub with the EHR establishing a patient context, see messages 1 through 7 in the below sequence diagram.
```

Comment:
Thank you for the clarification.  Stylistically I added a comma after the authorized for consistency.